test/v2/mocks/services/MockListBsasService.scala

```diff
@@ -34,10 +34,10 @@ trait MockListBsasService extends MockFactory{
 
   object MockListBsasService{
 
-    def listBsas(requestData: ListBsasRequest): CallHandler4[ListBsasRequest, HeaderCarrier, ExecutionContext, EndpointLogContext, Future[Either[ErrorWrapper, ResponseWrapper[ListBsasResponse[BsasEntries]]]]] = {
+    def listBsas(requestData: ListBsasRequest): CallHandler5[ListBsasRequest, HeaderCarrier, ExecutionContext, EndpointLogContext, String, Future[Either[ErrorWrapper, ResponseWrapper[ListBsasResponse[BsasEntries]]]]] = {
```

Comment:
Same here. Please use **CallHandler** instead of CallHandler5 to shorten the line. Thanks